packages/std-contracts/src/libraries/LibStorage.sol

```diff
@@ -0,0 +1,43 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import { IUint256Component } from "solecs/interfaces/IUint256Component.sol";
+import { IWorld } from "solecs/interfaces/IWorld.sol";
+import { getSystemAddressById, getAddressById } from "solecs/utils.sol";
+
+uint256 constant worldID = uint256(keccak256("mud.world"));
+uint256 constant componentsID = uint256(keccak256("mud.components"));
```

Comment:
do we still need these?

Reply:
Removed in [16e2033](https://github.com/latticexyz/mud/pull/264/commits/16e203374934ad3df1115076c0083939d69a1060)